src/java/picard/sam/ValidateSamFile.java

```diff
@@ -185,4 +195,18 @@ protected int doWork() {
 
         return result ? 0 : 1;
     }
+
+    @Override
+    protected String[] customCommandLineValidation() {
+        if ((!VALIDATE_INDEX  && INDEX_VALIDATION_STRINGENCY != IndexValidationStringency.NONE) ||
+            (INDEX_VALIDATION_STRINGENCY == IndexValidationStringency.NONE && VALIDATE_INDEX)) {
```

Comment:
to be consistent with the previous line, could you put `VALIDATE_INDEX` before `INDEX_VALIDATION_STRINGENCY == IndexValidationStringency.NONE`?